src/modules/editor/utils.ts

```diff
@@ -213,6 +214,29 @@ export function mergeWearables(avatar: Wearable[], apply: Wearable[]) {
   return Object.values(wearables)
 }
 
+export function patchWearables(wearables: Wearable[]) {
```

Comment:
Would you mind adding a test for this new function?

Reply:
done ✅